gen-interface.sh

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash
+
+set -e
+
+NEBULA_THIRDPARYTY_HOME=
```

Comment:
You can add the link that how to install third-party.

Reply:
The readme has note it.